src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -405,10 +404,6 @@ public void hoverClickAndViewPhotoOnTableCell(int questionBodyIndex, int tableRo
         
         waitForElementPresence(By.cssSelector(".popover-content"));
         
-        WebElement popoverContent = browser.driver.findElement(By.className("popover-content"));
-        
-        waitForElementVisibility(popoverContent);
-        
```

Comment:
I see. What should we do here (or anywhere else) if we want to ensure the popover works?